activity_browser/application.py

```diff
@@ -1,9 +1,9 @@
 # -*- coding: utf-8 -*-
 import logging
 
-from .controllers import controllers
-from .layouts import MainWindow
-from .logger import ABHandler
+from activity_browser.controllers import CONTROLLERS
+from activity_browser.layouts.main import MainWindow
+from activity_browser.logger import ABHandler
```

Comment:
You clearly explain the what in the PR comment, but not the why... what does changing from relative to absolute imports do here?

Reply:
The why is mostly clarity for other people/developers, absolute imports make it easier to follow where methods/classes come from. And with further refactoring it should be simpler to move code around without having to continuously adjust the import statements.